File: Практика - Timus Online Judge/1877/1877.cpp
#include <iostream>

using namespace std;

int main()
{
	int codeLock_1, codeLock_2;

	cin >> codeLock_1 >> codeLock_2;

	if (codeLock_1 % 2 == 0 || codeLock_2 % 2 != 0)
		cout << "yes" << endl;
	else
		cout << "no" << endl;

	return 0;
}

File: Практика - Timus Online Judge/1877/1877.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.InteropServices;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp1
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int codeLock_1, codeLock_2;

            codeLock_1 = Convert.ToInt32(Console.ReadLine());
            codeLock_2 = Convert.ToInt32(Console.ReadLine());

            if (codeLock_1 % 2 == 0 || codeLock_2 % 2 != 0)
                Console.WriteLine("yes");
            else
                Console.WriteLine("no");

            // string input = Console.ReadLine();
        }
    }
}
